src/Tribe/Repository.php

```diff
@@ -1562,7 +1562,7 @@ public function set_formatter( Tribe__Repository__Formatter_Interface $formatter
 	 * Filters the query to only return posts that are related, via a meta key, to posts
 	 * that satisfy a condition.
 	 *
-	 * @param string|array $meta_keys One ore more `meta_keys` relating the queried post type(s)
+	 * @param string|array $meta_keys One or more `meta_keys` relating the queried post type(s)
 	 *                                to another post type.
 	 * @param string       $compare   The SQL comparison operator.
 	 * @param string       $field     One (a column in the `posts` table) that should match
```

Comment:
What about flipping the position of the second and third parameters?

This way the signature would become:

```php
public function where_meta_related_by( $meta_keys, $field = null, $compare = 'EXISTS', $values = null )
```

Allowing for more readable calls like:

```php
$r->where_meta_related_by( '_review', '_short_line', '=', 'Would read it again!' );

// It has a review.
$r->where_meta_related_by( '_review' )

// Find books w/ reviews with a short line.
$r->where_meta_related_by( '_review', '_short_line', 'EXISTS' );

// Find books w reviews w/o a short line.
$r->where_meta_related_by( '_review', '_short_line', 'NOT EXISTS' );

// Find books w/o reviews.
$r->where_meta_related_by( '_review', null, 'NOT EXISTS' );
```

The last example is the one that would become less "elegant" yet, I think, would keep being readable. I think the change would make the method more readable in its most common use.

This is, but, a personal opinion and I would respect a decision in the existing direction.

Reply:
I think you meant this comment about `where_meta_related_by_meta()`, not `where_meta_related_by()`, right?

We kept the same ordering for `where_meta_related_by_meta()` so it wouldn't differ too much.